sdgym/benchmark.py

```diff
@@ -720,86 +720,52 @@ def _create_instance_on_ec2(script_content):
     print(f'Job kicked off for SDGym on {instance_id}')  # noqa
 
 
+# Import the REaLTabFormer synthesizer
+from realtabformer import REaLTabFormer
```

Comment:
This doesn't exist, create a file in `sdgym.synthesizers` and import it from there.